doc/conf.py

```diff
@@ -56,32 +55,12 @@
 automodapi_toctreedirnm = "code/api"
 automodsumm_inherited_members = True
 
+copybutton_prompt_text = r">>> |\.\.\. |\$ |In \[\d*\]: | {2,5}\.\.\.: | {5,8}: "
+copybutton_prompt_is_regexp = True
 
 intersphinx_mapping = {"https://pennylane.ai/qml/": None}
 mathjax_path = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
-
-from glob import glob
-import shutil
-import os
-import warnings
-
-# sphinx_gallery_conf = {
-#     # path to your example scripts
-#     'examples_dirs': '../examples',
-#     # path where to save gallery generated examples
-#     'gallery_dirs': 'tutorials',
-#     # build files that start 'pennylane_run'
-#     'filename_pattern': r'pennylane_run',
-#     # first notebook cell in generated Jupyter notebooks
-#     'first_notebook_cell': "%matplotlib inline",
-#     # thumbnail size
-#     'thumbnail_size': (400, 400),
-# }
-
-# Remove warnings that occur when generating the the tutorials
-warnings.filterwarnings("ignore", category=UserWarning, message=r"Matplotlib is currently using agg")
-warnings.filterwarnings("ignore", category=FutureWarning, message=r"Passing \(type, 1\) or '1type' as a synonym of type is deprecated.+")
-warnings.filterwarnings("ignore", category=UserWarning, message=r".+?Compilation using quilc will not be available\.")
```

Comment:
Did we get these working?

Reply:
These were warning filters for Sphinx-Gallery; we used to build the demos from within the PL documentation. Eventually we spun this out into its own repo, PennyLaneAI/qml. This is a remnant, I was surprised it was still there!